### DISCUSSION TITLE
[Bug] Inability to focus a window that does not use a headerbar

### DISCUSSION THREAD

**Discussion by toudi** [BuddiesOfBudgie/budgie-desktop#349] [General]
### Description

I discovered this recently when I installed budgie desktop on opensuse tumbleweed. Initially I couldn't figure out the reason and then I started comparing packages between fedora 37 and tumbleweed and I realized that what was different was in fact mutter. then I started googling some more and I discovered this bug report:

I'm only mentioning this because fedora 38 is about to be released and I think it bumps mutter release as well.

If you already know about this issue (though I haven't been able to find it here on github) feel free to close this bug report

### Budgie version

10.7.1

### Operating System

Opensuse Tumbleweed

### Steps to reproduce the issue

1. Open two windows which does not use a headerbar. For instance vs codium or chromium or virt-manager
2. Try to select the window by clicking on it's title bar

It only works with headerbar-enabled applications - like for instance budgie control center or gnome calculator

### Actual result

You cannot focus the window. You can only focus the window by clicking on it's content area

### Expected result

Being able to focus the window

### Additional information

https://gitlab.gnome.org/GNOME/mutter/-/issues/2660

**Comment by serebit** [comment]
Migrated this to a Discussion. We don't track third-party bugs in our own Issues, but this is still worth keeping as a reference for people who run into this problem.

**Comment by samlane-ma** [comment]
Also reported here:

https://gitlab.gnome.org/GNOME/mutter/-/issues/2727

As a workaround, certain things can be run with the environment variable GTK_CSD=1 to mitigate this.
For example, running `GTK_CSD=1 virt-manager` will get you a clickable title bar at the expense of forcing client side decorations. Doesn't work with everything.

I don't really like it, its  bandaid over an ugly issue, but there you have it.
